src/index.html

```diff
@@ -7,6 +7,248 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__content">
+        <img src="/images/logo.png" alt="MOYO Logo" class="header_logo">
+      </a>
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__li">
+            <a href="#part1" class="header__link is-active">APPLE</a>
+          </li>
+          <li class="header__li">
+            <a href="#part2" class="header__link">SAMSUNG</a>
+          </li>
```

Comment:
Нет отступов межде элементами блока, сложно различать. Я бы добавила отступы: 
```suggestion
          <li class="header__li">
            <a href="#part1" class="header__link is-active">APPLE</a>
          </li>
          
          <li class="header__li">
            <a href="#part2" class="header__link">SAMSUNG</a>
          </li>
```
И так по всему файлу. Но это, наверное, кому как удобно.